client/src/components/screens/PatientSearch/components/PatientTable.tsx

```diff
@@ -1,202 +1,196 @@
+  columnsReset: () => void;
+  size: number;
+  page: number;
 }
 
-const PatientTable: React.FC<Props> = ({
-  searchProps,
-  defaultVisibleColumns,
+type Row = {
+  birthDate: string;
+  familyId: string;
+  fetus: boolean;
+  firstName: string;
+  id: string;
+  lastName: string;
+  organization: string;
+  ramq: string;
+  request: string;
+  gender: string;
+};
+
+const TextCell = (value: string) => (
+  <Cell className="cellValue">
+    <Typography.Text ellipsis>{value}</Typography.Text>
+  </Cell>
+);
+
+const extractOrganization = (patient: PatientData) =>
+  patient?.organization.name || patient?.organization.id.split('/')[1];
+
+const makeRows = (rawResult: PatientData[]) =>
+  (rawResult || [])
+    .filter((currentPatientData: PatientData) => currentPatientData?.organization)
+    .map((currentPatientData: PatientData) => ({
+      birthDate: currentPatientData.birthDate ?? '--',
+      familyId: currentPatientData.familyId ?? '--',
+      fetus: currentPatientData.fetus ?? '--',
+      firstName: currentPatientData.firstName ?? '--',
+      gender: intl
+        .get(`screen.patientsearch.${currentPatientData.gender.toLowerCase()}`)
+        .defaultMessage('--'),
+      id: currentPatientData.id ?? '--',
+      lastName: currentPatientData.lastName.toUpperCase() ?? '--',
+      organization: extractOrganization(currentPatientData) ?? '--',
+      ramq: currentPatientData.ramq ?? '--',
+      request: currentPatientData?.requests?.length ? `${currentPatientData.requests.length}` : '0',
+    }));
+
+const makeColumns = (rawData: Row[], goToPatientPage: (patientId: string) => void) => [
+  {
+    key: 'patientId',
+    label: 'screen.patientsearch.table.id',
+    renderer: (rowIndex: number) => (
+      <Cell className="cellValue">
+        <Button
+          className="button link--underline"
+          onClick={() => goToPatientPage(rawData[rowIndex]?.id)}
+        >
+          {rawData[rowIndex].id}
+        </Button>
+      </Cell>
+    ),
+  },
+  {
+    key: 'ramq',
+    label: 'screen.patientsearch.table.ramq',
+    renderer: (rowIndex: number) => TextCell(rawData[rowIndex]?.ramq),
+  },
+  {
+    key: 'lastName',
+    label: 'screen.patientsearch.table.lastName',
+    renderer: (rowIndex: number) => (
+      <Cell className="cellValue">
+        <div className="patients-table__cell-container">
+          <p>{rawData[rowIndex]?.lastName?.toUpperCase()}</p>
+          {rawData[rowIndex]?.fetus && (
+            <Tooltip title={intl.get('screen.patient.table.fetus')}>
+              <img
+                alt={intl.get('screen.patient.table.fetus')}
+                className="patients-table__fetus-icon"
+                src="/assets/icons/patient-fetus.svg"
+              />
+            </Tooltip>
+          )}
+        </div>
+      </Cell>
+    ),
+  },
+  {
+    key: 'firstName',
+    label: 'screen.patientsearch.table.firstName',
+    renderer: (rowIndex: number) => TextCell(rawData[rowIndex]?.firstName),
```

Comment:
For some reason, I could not do: `renderer: (rowIndex: number) => <TextCell value={rawData[rowIndex]?.firstName} />`. I wasted too much time trying to make that work, so...